client/webpack.config.js

```diff
@@ -165,7 +165,7 @@ const webpackConfig = {
 		removeEmptyChunks: ! isDesktop,
 		splitChunks: {
 			chunks: 'all',
-			name: !! ( isDevelopment || shouldEmitStats ),
+			...( isDevelopment || shouldEmitStats ? {} : { name: false } ),
```

Comment:
https://webpack.js.org/plugins/split-chunks-plugin/#splitchunksname

`name: true` is makes webpack unhappy, it has to be `name: <function>` or `name: false`